jest.config.js

```diff
@@ -1,6 +1,7 @@
 const testsForPackage = (packageName) => `<rootDir>/packages/${packageName}/**/*.test.[jt]s?(x)`
 
 module.exports = {
+  roots: ['<rootDir>/packages'],
```

Comment:
According to the CI run, this doesn't prevent a lot of jest-haste-map collisions being output: https://buildkite.com/bugsnag/bugsnag-js/builds/3362#12b5bdbe-8113-4fa1-ba74-7d2ff119670d/122-128

We could give all those package.jsons unique names, but maybe there is some more tweaking of jest config to be done that would prevent it from looking there…

Reply:
I've sorted this now. roots needed setting for every jest project